teachertool/src/components/HeaderBar.tsx

```diff
@@ -12,10 +13,10 @@ export const HeaderBar: React.FC<HeaderBarProps> = () => {
 
     const getOrganizationLogo = () => {
         return (
-            <div className="ui item logo organization">
+            <div className={css["org"]}>
```

Comment:
Not quite following what this does. How is it different from just `className="org"`? Is it purely a intellisense/refactoring/maintainability thing, or does it have different functionality?

Reply:
This is how module css classes are referenced. Higher up in the file you'll see the associated import:

```ts
import css from "./styling/HeaderBar.module.scss";
```